features/diversity/diversity_landing page.feature

```diff
@@ -0,0 +1,12 @@
+@wip @diversity @javascript
+
+Feature: Diversity landing page
+
+Scenario: Successfully starts diversity questionnaire
+  Given I am on the diversity landing page
+  When I click on the start now button
+  Then I should be taken to the type of claim page
+
+Scenario: Timeout message
+  Given I am on the diversity landing page
+  Then I should see timeout message
```

Comment:
I should change this to 'Then I should see please note warning message about the questionnaire timeout'
Otherwise it could referring to the error message after the timeout has happened.